README.md

```diff
@@ -1,29 +1,34 @@
-# VoicearchiveClient
+# VoiceArchive Client Gem [![Build Status](https://travis-ci.com/ArnsboMedia/voicearchive_client.svg?branch=master)](https://travis-ci.com/ArnsboMedia/voicearchive_client)
 
-TODO: Write a gem description
+VoiceArchive API client gem, for use i Ruby projects. The projects implements some of the exposed API endpoint, from [Arnsbo Media Engine](https://github.com/ArnsboMedia/systemet).
 
-## Installation
+## Requirements
 
-Add this line to your application's Gemfile:
+- [Ruby](https://www.ruby-lang.org/en/news/2014/11/13/ruby-1-9-3-p551-is-released/) 1.9.3
 
-    gem 'voicearchive_client'
+## Installation
 
-And then execute:
+```shell
+$ scripts/setup
+```
 
-    $ bundle
+## Usage
 
-Or install it yourself as:
+Run `irb` to play around with the Gem:
```

Comment:
Shouldn't this be `scripts/server` instead of `irb`?
Or should we even have the `scripts/server` script. Why not simply use the `scripts/console` script?